cli/src/semgrep/core_runner.py

```diff
@@ -531,21 +528,6 @@ def _add_match_times(
             }
             profiling_data.set_file_times(Path(t.path), rule_timings, t.run_time)
 
-    def get_files_for_language(
-        self, language: Language, rule: Rule, target_manager: TargetManager
-    ) -> List[Path]:
-        try:
-            targets = target_manager.get_files_for_rule(
-                language, rule.includes, rule.excludes, rule.id
-            )
-        except _UnknownLanguageError as ex:
```

Comment:
get_files_for_rule never raised this exception. This exception is not raised anywhere in the codebase